.gitlab-ci.yml

```diff
@@ -399,7 +399,7 @@ check_sphinx:
   script:
     - sed -i 's/ or "DISPLAY" in os.environ/ or True/' ${CI_PROJECT_DIR}/src/python/espressomd/visualization.pyx
     - cd ${CI_PROJECT_DIR}/build
-    - make -t && make sphinx
+    - make -t && rm src/python/espressomd/visualization.* && make sphinx
```

Comment:
why do we need this?

Reply:
We have to change the code in a Cython file, otherwise it cannot be imported in python in a docker container. This was done by a sed expression. Then `make -t` made everything up-to-date, and CMake printed the splash screen with the project details and rebuilt the modified Cython file. But with the new container, CMake doesn't realize the Cython file was modified after `make -t` and doesn't rebuild it, causing this [CI error](https://gitlab.icp.uni-stuttgart.de/espressomd/espresso/-/jobs/164414). If you delete the .so files then do `make -t`, CMake won't rebuild it either, causing an import error. So we delete the file after `make -t` to trigger the rebuild.